testsuite/domain/src/test/resources/base-server-config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<server xmlns="urn:jboss:domain:20.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:jboss:domain:20.0 wildfly-config_20_0.xsd">
+<server xmlns="urn:jboss:domain:community:20.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:jboss:domain:community:20.0 wildfly-config_community_20_0.xsd">
```

Comment:
I think our last agreement was not to have community schemas in these configuration files. If we have specific community stability level tests, we should reload the server into that stability level before running those tests. Otherwise, we will need to convert these configuration files back into default schema when creating the product branches and we want to avoid that.

Reply:
@yersan FYI we have the following Jira issue to come back and follow up on this https://issues.redhat.com/browse/WFCORE-6972 this is why the commit message says it is a temporary change.  I have bumped the follow up Jira issue to Critical so we can come back to these as these configuration files already contain community schema references so this is not something new.

Added to this this is becoming a real rabbit hole which is intertwined by tests @ehsavoie is also evolving further so I think we need to get this PR in and Emmanuel's PR in then come back and fix up this testsuite.

There is also further information on https://github.com/wildfly/wildfly-core/pull/6153 regarding the types of failure we see if we try to just use the default stability level.

Reply:
FYI this was my closing conclusion on the chat thread on the 2nd Sept:

> On the original discussion for this thread, as these tests are also being refactored for their own feature promotion I think I am going to take a step back on that and temporarily just add another community namespace to the configuration schema just so I can progress with my feature.
> 
> Once my feature and Emmanual's promotion is ready we can look at how to combine at the end to complete some of the thoughts here.

Reply:
> Added to this this is becoming a real rabbit hole which is intertwined by tests @ehsavoie is also evolving further so I think we need to get this PR in and Emmanuel's PR in then come back and fix up this testsuite.

Ok, got it, I mistakenly thought that the changes here were to accommodate the `ManagementInterfaceResourcesTestCase`, but they are just because after incorporating the community schema we need to bypass the flaws in the `ManagementReadXmlTestCase` test case, which is not able to work with schemas in default stability level.

However, independently of that, since this `ManagementInterfaceResourcesTestCase` now is testing changes that are only in community stability level, should not be already prepared to reload the server into the expected stability level? That would make it to skip these tests if the community stability level is not available and the server was provisioned to default stability level.

I am fine if we postpone this to later, but just wanted to crosscheck if you agreed with that plan.

Reply:
FYI I just added an inline comment where the test checks the attributes are available to decide if that part of the test should run / be skipped.